config/users.php

```diff
@@ -62,6 +62,18 @@
         //
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Password Requirements
+    |--------------------------------------------------------------------------
+    |
+    | When setting a new password the password will be validated against
+    | these additional rules to prevent any weak passwords for users.
+    |
+    */
+
+    'password_validation' => [],
```

Comment:
I wonder if it might be worth adding in the default password validation rules here so the user knows what they are extending upon?

Reply:
I didn't want to do that because removing them could be catastrophically.
But I am open to suggestions :-)

Also the `confirmed` is not always needed

---

And adding something like `min:6` what would make sense IMHO but could be a breaking change

Reply:
> I didn't want to do that because removing them could be catastrophically.

What do you mean, as in, stuff would fail?

Reply:
I did not test if but removing `confirmed` could lead to user input errors and unknown passwords.
Removing `required` could lead to empty password being accepted I suppose

Reply:
Hmm, that's a good shout. I guess that's probably something for the Core Team to decide on...